srcStatic/Log/LogDebug.h

```diff
@@ -3,7 +3,7 @@
 #include "LogDestination.h"
 
 
-class LoggerDebug : public LogDestination
+class LogDestinationDebug : public LogDestination
```

Comment:
This rename looks like a search/replace error.

Reply:
It was on purpose. Would you prefer a different name?

Reply:
Hmm, I see now. That is unfortunate. We can't have a class with the same name as a function. Though if we rename either one of them to include "Destination", we've introduced an asymmetry to the naming in our API. I kind of would like to rename this, though we may need to be creative. What if we changed the "Debug" part? Is there something more descriptive we could use there? Maybe `LogAttachedDebugger` for the class name?